### DISCUSSION TITLE
Some ideas for the revenue model and how to make this sustainable

### DISCUSSION THREAD

**Discussion by Oshyan** [PullDating/Org#2] [Revenue and Funding]
To my mind one of the key considerations that needs to be discussed, if not resolved, early on is how this project will be able to pay for the strictly necessary costs of providing baseline services to a dating user base. If we assume _for now_ that there are no paid positions in this org, then the primary costs will be hosting of the back-end in a secure, reliable, and high performance way. 

Because dating apps are image-heavy, and potentially (more recently) even *video* heavy, there are non-trivial storage space and bandwidth requirements, at the least. Because the success of a dating app is closely tied to the size of its userbase, achieving a large scale is also critical. Even if we assumed we could host things for free or cheap initially, once we tried to scale beyond e.g. 100k users (a tiny amount when distributed around the world by location), we would almost certainly need to begin paying $1000+/mo just for hosting. But this is absolutely something we should try to research so we know roughly what our costs might be as we scale.

- [ ] Research and/or estimate data usage and associated hosting costs for dating app profiles and interactions

# Revenue not Profit
It seems unwise to try to make this work on donations alone (though that's an option, see below). I think we all also agree that a core problem with almost all existing dating apps is that the need for profit and the methods by which those profits are generated puts the incentives of the app publisher in direct opposition to the users. Users most often want to meet someone great and stop having to use the app. Publishers meanwhile need people to keep using it to make money to sustain its continued service. 

We need to avoid this misalignment in order for this project to succeed, yet we also need a reliable source of funds to sustain baseline costs of hosting, at the very least. And arguably more funds may be valuable, if not vital, for long-term viability, e.g. paying core contributors so they can actually focus their time on building a legitimately competitive, excellent platform/app/experience.

# Some Revenue Concepts
I have some ideas for this which I'll try to outline briefly here. But obviously these are just some of the possibilities. The intent of this is just to *start* the discussion. I am looking forward to seeing how other people feel we could solve this core challenge!

## Donations
This is basically the Wikipedia Model - develop a valuable service then do donation drives. I'm not a Wikipedia historian, but my basic understanding is it was privately funded (by Bomis) for the initial years, then became user-supported. But it was not for several years after inception that formal fundraising drives, etc. started to happen. So there's that initial period where, even if you *are* soliciting donations, the income may not be enough to cover costs.

The basic approach would be to stay lean and do whatever we could do host the basic dating profile content and services until we could build a decent product (MVP, basically) and then solicit donations from the users. Obviously we don't have to wait to start asking for that, but people will probably be more likely to contribute if they can see clear value and quality in the service that they're donating to. 

The main issue with this approach overall is lack of reliable income. You just don't know how much contribution you'll get. And there can be outsized negative feedback effects where e.g. one month you don't get enough money to cover hosting costs, so you have to let go of a server or two, the service becomes less reliable and more people jump ship/become less willing to donate, etc. Of course it could go the other way for some percentage of people too: things slow down due to lack of money to pay for servers, we use that as real evidence of legitimate need for donations, and some people do donate. The point is it's hard to predict, and that kind of sucks.

### Kickstarter or similar initial funding

One possible way to help with the initial costs, as well as potentially create a reserve fund for future shortfalls in a donation model, would be an initial lump sum funding period like a Kickstarter (but not necessarily using Kickstarter itself). There are pros and cons to this, but one pro is that it can be a way to both generated sufficient interest before a real launch, and to help *gauge* interest by number of supporters, etc. It's not a perfect indicator, but it's one way to tangibly get some press and other attention, etc.

### Transparency and use of funds
An important aspect of this approach, and of the project as a whole, is to be really clear about what the costs of the service are and what the funds are actually being spent on.

## Charge everyone a low _monthly_ fee
OK, I know, but hear me out... First of all, with *any* fee-based approach two things should probably be in place:
1. Clear, consistent, and open communication for how we use the funds
2. We have some mechanism to sustainably allow people who lack the funds to participate

With that in mind, the basic idea here that literally everyone is a paying member, but at a very low cost, like $1/mo i.e. $12/yr. This is hopefully sustainable for *most* potential daters, if you consider users of current apps. If you can afford a smart phone you can probably afford this. Whether people *want* to pay is another question that would need to be discussed and tested. Obviously there are a possibly large subset of people who say they'll never pay for any dating app. That may or may not be a portion of people worth losing as customers.

There are advantages to this approach, though. Chief among them is it probably takes care of a lot of potential scammers, spammers, etc. Having "skin in the game" also hopefully makes people a bit more invested in using the service in good faith in general and not taking it all for granted.

There are clearly questions around what amount is strictly necessary to support the costs of hosting. Ideally we'd figure out what that number is and charge that, or a tiny bit more to have some reserves for fluctuations or other issues. Again all of this would be communicated transparently. And likewise I think that being a non-profit or B Corp or something might help people feel more inclined to pay, knowing where the money is going, etc.

## Charge everyone a low _1-time_ fee
Same idea as above, except a 1-time fee. I'm not sure this is any better, per se, and it has some concerns as far as predictability of revenue. But one of the interesting things about a dating app is that almost everyone is a potential customer, and you are constantly having new possible customers (people coming of age). So once your reach/awareness is big enough, there might not actually be that much difference between a monthly payment model and a one-time payment model. Same pros and cons apply as above I think, more or less.

## Distributed Hosting

There are multiple possibilities here, but distributed apps are generally more complicated to develop and maintain, at least in certain respects. If it works, though, it could be a clean solution to most of the costs in consideration here.

A reasonably simple model might draw inspiration or even code from existing distributed social network platforms and protocols, like ActivityPub, Mastodon, Diaspora, etc. People could host their own servers at will, we'd provide and maintain the source code for that via Github. Servers could connect to each other, but with optional restrictions each server admin could control (e.g. you could, if desired, have a fairly closed server focusing on a particular niche of people, like those who are demisexual or whatever). The key to making this work, aside from the technical aspects, is a really good server discovery mechanism, or even a semi-automated one where choosing a "server" does not become a notable initial barrier to just getting into the app and finding people quickly. You want to give people choice, but not *require* any more choices than necessary.

### Web 3/Blockchain/etc.
The whole concept of a distributed approach is very interesting and also potentially opens up a whole other area of possibility that may or may not have appeal to people, things like blockchain identity, IPFS and Filecoin for storage, DAOs, etc. But it might depend in part on whether people with expertise in those areas want to join the team. Of course there's a bit of chicken and egg there because we could also be actively recruiting for such people if we felt it was a strong possible way to make this sustainable. So I do think we should decide for ourselves to what degree this seems valuable.

# Other Related Considerations
The revenue model also connects in potentially significant ways with multiple other highly important aspects of the project:
* Company/organizational model and formation
* Governance
* Tech stack (as relates to e.g. distributed vs. not, at the least)

# Share your thoughts!
I know this was *a lot*. But I hope you'll agree that is one of a few really significant things to discuss and decide on as a group before we get too far along. There's not much point building a cool app if we'll run out of funding to continue building or hosting it. Likewise the way we communicate with prospective collaborators as well as users should be heavily informed by our choices around revenue model, organizational structure (if you're a non-profit, collective, or B corp, it will affect how people feel about e.g. donating money or paying for services, etc.), etc. 

So with that in mind, I look forward to each of your thoughts and ideas on how to make this work sustainably!

**Comment by Oshyan** [comment]
As a follow-up to the Kickstarter idea, Alovoa tried that already and failed:
https://www.kickstarter.com/projects/nonononoki/alovoa-the-private-dating-app-for-you/
https://www.kickstarter.com/projects/nonononoki/alovoa-the-private-dating-app-for-you/comments

It's worth trying to understand why, although it may just not be that interesting an idea to that kind of audience. That said I think a collective effort, rather than efforts of a single person. Some of the messaging around Alovoa's campaign kind of made it feel very "one man show" and smaller scale, not really aiming for a big vision, etc. Crafting a compelling Kickstarter and marketing it is a skill in itself, and one we should take seriously if we consider that approach. At the least I think we can say the Alovoa dev does not seem to have taken it that seriously and kind of just hoped/expected the idea of a "private"/ethical app in itself would be interest people enough to fund it.

We might also want to get in touch with him to ask him about his experience, what he did to market the kickstarter, etc.

**Comment by flo-kei** [comment]
I think charging everyone a monthly or one-time fee sounds like the most reliable solution. 

Considering that this service could finally give them what they're looking for, they might be willing to pay for this.
(we should probably run the numbers to figure out if a one-time fee covers our costs, since people might be more willing to pay once than forever (here i assume the one-time fee isn't particularly high))

However there are also quite a few people whom are not interested in spending money on an app like this.

We could also make the one-time payment optional but encourage users to donate by giving them incentives to do so, maybe even peer pressure (e.g. fancy backgrounds for those who paid it, everyone else's profile kinda looks the same:)
(if we opt for something like this, we should make sure the experience of those not willing to pay is not affected negatively)

or we could allow "skipping the fee" if the person truly cannot afford it, but then has to prove so? (and maybe also exclude students and the likes? - only if the fee is more than a few bucks)

Our app should not become an "ELITE DATING APP" only for folks who can afford to be on it.
(besides, this would probably eliminate the possibility of ever becoming mainstream)

everyone should be welcome here and we should not burden users more than absolutely necessary with hurdles like a paywall

**Reply by Oshyan** [reply]
Pretty much agree with everything you wrote here. The one-time payment approach is not necessarily my preferred one, but there are unique dynamics in a dating app that might make it more functional than it is in other software... potentially. But ongoing **low** fees just for continued use make the most sense: tie the costs to the active status of the user. When they're active, they pay, when they're not (deactivate account), they don't. 

We'd have to run the numbers, but with modern bulk server/data/storage pricing combined with modern video/image compression technologies, I think the baseline service running costs could be pretty low. This would allow us to keep the monthly fee extremely low, like $1-3/mo, maybe $5/mo at the very most. This is still far, far below *every* other fee-based dating app, and we would be providing a much better overall experience anyway, with no additional "pay to play" features like "Superlikes" or "Boosts", etc. Everything should be on an equal playing field.

Another advantage of the mandatory fee is anti-spam/scam effects. And it also potentially increases the "seriousness" of people's dating intent. One thing on many current dating apps that I think is toxic to the "culture" of each app in general is people being on there just for ego stroking or browsing hot profiles, without actually having a serious intent to connect in some way. I don't know what percentage of people this actually is, but it's definitely *some* percentage, and having a mandatory fee also helps address that.

I think we also in general need to recognize that not everyone *will* use this app, even if it might be better than the alternatives. You mention some people just aren't willing to pay. Well, yes, so they won't use the app. This is not a reason to make the app free in spite of the disadvantages of doing so. Certainly we should consider overall the trade-offs we make and whether we could in some way make it totally free, that *might* be ideal in a perfect world, but as long as we have service costs to deal with we'll need revenue from somewhere. And, as I said, there are other benefits of 

I don't think that we can reasonably accommodate "skip the fee" stuff. It would require time to validate the request, and that makes the app and staff a lot less "lean", which somewhat challenges the low overhead approach. It's a nice idea, but needs some practical way to address the time sink aspect. Maybe a "gift membership pool" could work as an alternative. See for example the Maximum Fun gift membership model:

> We’ve also set up a way for generous MaxFunsters who are in a good place financially to purchase a membership for an anonymous MaxFunster who can’t afford a membership this year. If you’d like to purchase one year’s worth of a $5 monthly membership for an anonymous MaxFunster, go to [maximumfun.org/join/#gift](https://maximumfun.org/join/#gift). If you would love to support your favorite MaxFun shows, but it’s just not in the budget this year, you can [add your name to the waiting list](https://forms.gle/R1UDagHcjACAsJsG8) of recipients.

I don't think we're in *any* danger of ever becoming "elite". We're talking about fees that are still way below literally any other commercial dating app (that I'm aware of).
